tests/helpers/s3Helper.ts

```diff
@@ -1,67 +1,75 @@
 /* eslint-disable @typescript-eslint/naming-convention */
+import { randSentence } from '@ngneat/falso';
 import {
-  CreateBucketCommandInput,
   CreateBucketCommand,
-  S3Client,
-  S3ClientConfig,
-  PutObjectCommand,
-  PutObjectCommandInput,
+  CreateBucketCommandInput,
   DeleteBucketCommandInput,
-  DeleteBucketCommand,
   DeleteObjectCommandInput,
+  PutObjectCommandInput,
+  DeleteBucketCommand,
+  S3Client,
+  PutObjectCommand,
   DeleteObjectCommand,
+  GetObjectCommand,
   ListObjectsRequest,
   ListObjectsCommand,
+  S3ClientConfigType,
+  GetObjectCommandInput,
 } from '@aws-sdk/client-s3';
-import { randSentence } from '@ngneat/falso';
-import { inject, injectable } from 'tsyringe';
-import { SERVICES } from '../../src/common/constants';
 import { S3Config } from '../../src/common/interfaces';
 
-@injectable()
 export class S3Helper {
   private readonly s3: S3Client;
 
-  public constructor(@inject(SERVICES.PROVIDER_CONFIG) protected readonly s3Config: S3Config) {
-    const s3ClientConfig: S3ClientConfig = {
-      endpoint: this.s3Config.endpointUrl,
-      forcePathStyle: this.s3Config.forcePathStyle,
+  public constructor(private readonly config: S3Config) {
+    const s3ClientConfig: S3ClientConfigType = {
+      endpoint: config.endpointUrl,
       credentials: {
-        accessKeyId: this.s3Config.accessKeyId,
-        secretAccessKey: this.s3Config.secretAccessKey,
+        accessKeyId: config.accessKeyId,
+        secretAccessKey: config.secretAccessKey,
       },
-      region: this.s3Config.region,
+      region: config.region,
+      maxAttempts: config.maxAttempts,
+      forcePathStyle: config.forcePathStyle,
```

Comment:
This class is for tests so there is no need to add this extra configurations (the default is false for this field). If you want to add this anyway, you should add also the tls field.

Reply:
ok